utils/src/main/kotlin/Extensions.kt

```diff
@@ -370,6 +370,12 @@ fun String.stripCredentialsFromUrl() =
         this
     }
 
+/**
+ * If this string starts with [prefix], return the string without the prefix, otherwise return [missingPrefixValue].
+ */
+fun String?.withoutPrefix(prefix: String, missingPrefixValue: String? = null): String? =
```

Comment:
The parameter for the default value could be a lambda, so that it is evaluated only if necessary.

Reply:
Done.